lib/_tls_wrap.js

```diff
@@ -608,6 +623,21 @@ TLSSocket.prototype._init = function(socket, wrap) {
       // Remover this listener since its no longer needed.
       this.removeListener('newListener', newListener);
     }
+
+    ssl.onkeylog = onkeylogclient;
+
+    // Only call .onkeylog if there is a session listener.
```

Comment:
do you mean "only if there is a keylog listener"?

Reply:
Ahh, the copy-paste :sweat_smile: Done!